OpenRA.Game/Graphics/Animation.cs

```diff
@@ -50,18 +51,22 @@ public Animation(World world, string name, Func<WAngle> facingFunc, Func<bool> p
 		}
 
 		public int CurrentFrame => backwards ? CurrentSequence.Length - frame - 1 : frame;
-		public Sprite Image => CurrentSequence.GetSprite(CurrentFrame, facingFunc());
+
+		public Sprite Image => CurrentSequence.GetSprite(CurrentFrame, facingFunc(), out rotation);
 
 		public IRenderable[] Render(WPos pos, in WVec offset, int zOffset, PaletteReference palette)
 		{
 			var tintModifiers = CurrentSequence.IgnoreWorldTint ? TintModifiers.IgnoreWorldTint : TintModifiers.None;
 			var alpha = CurrentSequence.GetAlpha(CurrentFrame);
-			var imageRenderable = new SpriteRenderable(Image, pos, offset, CurrentSequence.ZOffset + zOffset, palette, CurrentSequence.Scale, alpha, float3.Ones, tintModifiers, IsDecoration);
+			var imageRenderable = new SpriteRenderable(Image, pos, offset, CurrentSequence.ZOffset + zOffset, palette,
+													   CurrentSequence.Scale, alpha, float3.Ones, tintModifiers, IsDecoration, rotation,
+													   0);
```

Comment:
To be honest, I feel this isn't worth putting on an extra line just for readability/optics (assuming that was the reason).

Reply:
Actually the reason is it would be too long and require horizontal scrolling otherwise, the readability is purely an unintended side effect:

![image](https://user-images.githubusercontent.com/1495872/151649621-65f06658-8d2f-4ae4-87ab-47402e8ef81e.png)